validator/remote/src/main/java/tech/pegasys/teku/validator/remote/FailoverValidatorApiHandler.java

```diff
@@ -78,6 +79,11 @@ public FailoverValidatorApiHandler(
     this.validatorLogger = validatorLogger;
   }
 
+  @Override
+  public SafeFuture<Optional<SyncingStatus>> getSyncingStatus() {
+    return tryRequestUntilSuccess(ValidatorApiChannel::getSyncingStatus);
+  }
```

Comment:
This seems suspicious - wouldn't it be important to know which node wound up returning the syncing status? This may be another indication that we don't actually want to add this method to the `ValidatorApiHandler` interface.